README.md

```diff
@@ -74,21 +50,81 @@ Note that you can also pass the API key as an argument of the `Kili` initializat
 ```python
 kili = Kili(api_key='MY API KEY')
 ```
+Here is a sample of the operations you can do with the Kili client:
+### Creating an annotation project
+```python
+project_json = {
+    "jobs": {
+        "CLASSIFICATION_JOB": {
+            "mlTask": "CLASSIFICATION",
+            "content": {
+                "categories": {
+                    "RED": {"name": "Red"},
+                    "BLACK": {"name": "Black"},
+                    "WHITE": {"name": "White"},
+                    "GREY": {"name": "Grey"}},
+                "input": "checkbox"
+            },
+            "required": 0,
+            "isChild": False,
+            "instruction": "Color"
+        }
+    }
+}
+project = kili.create_project(
+        title='Color classification',
+        description='Project ',
+        input_type='IMAGE',
+        json_interface=json_interface
+)['id']
+```
+### Importing data to annotate
+```python
+assets = [
+    {
+        'externalId': 'example 1',
+        'content': 'https://images.caradisiac.com/logos/3/8/6/7/253867/S0-tesla-enregistre-d-importantes-pertes-au-premier-trimestre-175948.jpg',
+        'metadata': {"imageUrl": "https://i.pinimg.com/474x/f5/5e/95/f55e95f5677d2cb3bea922599667b6db--tesla-logo-car-logo-design.jpg",
+                     "text": "Tesla Company",
+                     "url": "https://www.tesla.com/fr_fr" },
+        'toBeLabeledBy':['test@kili-technology.com']
+    },
+    {
+        'externalId': 'example 2',
+        'content': 'https://img.sportauto.fr/news/2018/11/28/1533574/1920%7C1280%7Cc096243e5460db3e5e70c773.jpg',
+        'metadata': {"imageUrl": "https://upload.wikimedia.org/wikipedia/fr/thumb/e/e7/Logo_Porsche.svg/langfr-560px-Logo_Porsche.svg.png",
+                     "text": "Porsche Company",
+                     "url": "https://www.porsche.com/france/" },
+        'priority': 1
+    },
+    {
+        'externalId': 'example 3',
+        'content': './recipes/img/man_on_a_bike.jpeg',
+        'metadata': {},
+    },
+]
+
+external_id_array = [a.get('externalId') for a in assets]
+content_array = [a.get('content') for a in assets]
+json_metadata_array = [a.get('metadata') for a in assets]
+
+kili.append_many_to_dataset(
+        project_id=project_id,
+        content_array=content_array,
+        external_id_array=external_id_array,
+        json_metadata_array=json_metadata_array)
+```
+See the detailled example in this [notebooks](recipes/getting-started/getting_started-image_classification.ipynb).
 
```

Comment:
To have the full value proposition, I suggest:
- Creating an annotation project -> great, done
- Importing data to annotate -> great, done
- Importing predictions -> to add
- Exporting labels -> to add
- AutoML -> explain the concept and put a link to the repo, saying it's working in progress